services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -633,6 +669,11 @@ private Response putRelativeFile(EditorConfig config, String target, boolean ove
                      .entity("{\"error\": \"" + e.getMessage() + "\"}")
                      .type(MediaType.APPLICATION_JSON)
                      .build();
+    } catch (PermissionDeniedException e) {
+      return Response.status(Status.FORBIDDEN)
+                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
```

Comment:
Don't return direct exception message here (if it's only not ours one, already prepared for client/user side) but tells something clear an short and then use the same text in the LOG error/warn message. Apply to all such exception handlers.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/10/commits/36cfb2d370ee05b1aab1b09e471285054e5971e8